js/views/chatHeadsVw.js

```diff
@@ -40,6 +43,15 @@ module.exports = baseVw.extend({
 
   chatHeadClick: function(vw) {
     this.trigger('chatHeadClick', vw);
+    this.chatHeadsRemoveSelectStyle(vw);
+  },
+
+  chatHeadsRemoveSelectStyle: function(vw){
+    //remove any existing selected state
+    this.$('.js-chatHeadWrapper').removeClass('chatHeadSelected');
+    if(vw){
+      vw.$el.addClass('chatHeadSelected');
+    }
```

Comment:
I consolidated the functions to remove the select style from the chatHeads here.